src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,34 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    static final char CONSTATNT_1 = 'w';
+    static final char CONSTANT_2 = 'W';
```

Comment:
let's give more informative names to constants. And I really doubt that you need both of them